addons/hr_recruitment/models/hr_employee.py

```diff
@@ -9,6 +9,10 @@ class HrEmployee(models.Model):
 
     applicant_id = fields.One2many('hr.applicant', 'emp_id', 'Applicant')
 
```

Comment:
and we can override it here
```suggestion

def _get_partner_count_depends(self):
        return super()._get_partner_count_depends() + ['applicant_id']

```